tmux.go

```diff
@@ -115,6 +115,19 @@ func (tmux Tmux) SwitchClient(target string) error {
 	return tmux.commander.ExecSilently(cmd)
 }
 
+func (tmux Tmux) SessionName() (string, error) {
+
+	cmd := exec.Command("tmux", "display-message", "-p", "#S")
+	out, err := tmux.commander.Exec(cmd)
+	sessionName := out
```

Comment:
Do we really need this variable? Perhaps just return `out`?

Reply:
You're right. I cleaned the code